test/integration/test_interface.py

```diff
@@ -190,6 +190,68 @@ def test_multiple_inventories(project_fixtures):
     assert 'host_2' in stdout
 
 
+def test_default_project_dir(project_fixtures):
+    private_data_dir = project_fixtures / 'project_dir_test'
+
+    res = run(
+        private_data_dir=private_data_dir,
+        playbook='debug.yml',
+    )
+    stdout = res.stdout.read()
+
+    assert res.rc == 0, stdout
+    assert 'project' in stdout
```

Comment:
This will also match `my_project`. Best to be more explicit and use `In project` instead.